articles/faq/daily-limit-bigger-than-monthly.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "My daily limit is bigger than my monthly plan limit - which one is correct?"
+description: "In certain situations you may see in the SparkPost Application that your daily limit is larger than your current plan's monthly limit. In this situation the monthly limit takes precedence and you will be allowed to send to your monthly limit in a single day. "
+---
+
+In certain situations you may see in the SparkPost Application that your daily limit is larger than your current plan's monthly limit. For example if you have downgraded from a much larger plan or moved from one of our older plans. We leave this limit in place so if you upgrade you can use this larger limit without the need to request it through the application
```

Comment:
Maybe just `...you may see that your daily limit is larger than your monthly limit`? You could also link to the dashboard where these limits are shown: https://app.sparkpost.com/dashboard.

Reply:
I might also be explicit each time we say "limit". So always say "daily limit" or "monthly limit".

Reply:
`...without the need to request it.` <- shorter and with a full stop?